HACKING.md

```diff
@@ -35,9 +35,9 @@ This will start the builds in the background. Use oc logs -f build/<component> t
 follow the build logs, or use oc start-build --follow <component> to run the build 
 in the foreground and follow the build logs.
 
-In order to run a deployment with these images, you would process the
-[deployer template](deployer/deployer.yaml) with the `IMAGE_PREFIX`
-parameter pointing to the cluster registry location (e.g `IMAGE_PREFIX=172.30.90.128:5000/logs/`).
+In order to deploy logging with these images, you set the 
+[openshift-installer](https://github.com/openshift/openshift-ansible/tree/master/roles/openshift_logging) 
+inventory variable `openshift_logging_install_logging` to the cluster registry location (e.g `172.30.90.128:5000/logs/`).
```

Comment:
Is this correct?  I thought `openshift_logging_install_logging` was a boolean?

Reply:
correct, its a boolean